app/src/androidTest/java/org/sagebionetworks/research/mpower/AppOverviewStepUITests.java

```diff
@@ -0,0 +1,55 @@
+package org.sagebionetworks.research.mpower;
+
+import android.support.test.filters.LargeTest;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.*;
+
+@RunWith(AndroidJUnit4.class)
+@LargeTest
+@SuppressWarnings("PMD")
```

Comment:
The naming convention for test files is to end them with "Test" and start test methods with "test". This isn't required anymore with annotation driven JUnit, but a lot of tooling is still set up under these assumptions. If you named this class AppOverviewStepUITest you likely wouldn't need this PMD suppression. Should also pick up the FindBugs and Checkstyle exclusions